fixtures/case_entity.yaml

```diff
@@ -403,6 +403,7 @@ App\Entity\ResidentComplaintBoardCase:
     complainantPhone: 87654321
     complainantAddress: '@test-case-aarhus-1-complainant-address'
     leaseAddress: '@test-case-aarhus-1-lease-address'
+    sortingAddress: '@test-case-aarhus-1-lease-address->street @test-case-aarhus-1-lease-address->number , @test-case-aarhus-1-lease-address->postalCode @test-case-aarhus-1-lease-address->city'
```

Comment:
If you can use properties from objects fetched by `@`, you may be able to use `@test-case-aarhus-1-lease-address->__toString()` to more closely match how `sortingAddress` is actually set for real.

Reply:
They are actually not necessary at all since they are overwritten by `updateSortingAddress` in the specific case entities (`ResidentComplaintBoardCase` etc.).

Removed `sortingAddress` from `CaseEntity` fixtures.